tests/Doctrine/Tests/ORM/Functional/EntityRepositoryTest.php

```diff
@@ -1092,18 +1091,4 @@ public function testFindMultipleByNullValueInMultipleInCriteriaValues(): void
             self::assertContains($user, [$user1, $user2]);
         }
     }
-
-    public function testDeprecatedClear(): void
-    {
-        $repository = $this->_em->getRepository(CmsAddress::class);
-
-        if (class_exists(PersistentObject::class)) {
-            $this->expectDeprecationWithIdentifier('https://github.com/doctrine/orm/issues/8460');
-        } else {
-            $this->expectException(NotSupported::class);
-            $this->expectExceptionMessage(CmsAddress::class);
-        }
-
-        $repository->clear();
-    }
```

Comment:
This test covers the `PersistentObject` removal. We should keep it in some form.